### ISSUE TITLE
Combobox option doesn't have aria-selected set to true when opened

### ISSUE THREAD

**Issue by makopch-ms** [microsoft/fluentui#19264]
### Environment Information

- **Package version(s)**: v8
- **Browser and OS versions**: all

### Please provide a reproduction of the bug in a codepen:
v7: https://codepen.io/makopch/pen/LYyJONM?editors=1111
v8: https://codepen.io/makopch/pen/ExmeoLj?editors=1111

*Note* open console and wait 2 seconds after opening menu for it to dump the selected element there.

#### Actual behavior:

In v7, when the combobox is opened, the selected option has aria-selected=true w/o hovering
In v8, the item is never true
#### Expected behavior:

The selected item is true when opening the menu since it is the selected/checked item.

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No) Yes, with guidance

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable) Office Online

**Comment by makopch-ms** [comment]
@ecraig12345

**Comment by ecraig12345** [comment]
For reference, I intentionally made this change in #16331. @smhigley would appreciate your thoughts on correct handling. I also wonder if it needs to be different between single/multi select?

**Comment by ecraig12345** [comment]
For reference: [aria-checked](https://www.w3.org/TR/wai-aria/#aria-checked) vs [aria-selected](https://www.w3.org/TR/wai-aria/#aria-selected)

**Comment by smhigley** [comment]
Yup, it looks like we aren't correctly setting `aria-selected` for the current selected item (where the selected item is the item that is reflected in the value of the input, and has no relation to focus/hover states).

Checked vs. selected isn't applicable here, since checked/`aria-checked` is not a state that has any meaningful support on a combobox, and is not functionally different from `aria-selected` in this case.

**Comment by ecraig12345** [comment]
@smhigley Thanks for the feedback! After looking at your PR, I suspect the original issue that made me think aria-checked was needed was the general confusion in Combobox between selected and hover/focus states. Removing the select-on-hover/focus behavior seems like a better solution.
